<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTaskManagerClass</class>
 <widget class="QMainWindow" name="QtTaskManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTaskManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="hideFileButton">
      <property name="text">
       <string>Hide File</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="denyUserButton">
      <property name="text">
       <string>Deny Access</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="QTableWidget" name="processTable"/>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="cp_label">
        <property name="text">
         <string>Create New Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cp_button">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Action For Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pidLine">
        <property name="text">
         <string>Enter PID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="processActionBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="commit_button">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="forceUpdateButton">
      <property name="text">
       <string>Force Update</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="readonlyButton">
      <property name="text">
       <string>Enable Read-only</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
